pkgs/development/compilers/go/1.7.nix

```diff
@@ -116,7 +116,7 @@ stdenv.mkDerivation rec {
       })
     ];
 
-  SSL_CERT_FILE = "${cacert}/etc/ssl/certs/ca-bundle.crt";
+  NIX_SSL_CERT_FILE = "${cacert}/etc/ssl/certs/ca-bundle.crt";
```

Comment:
I hope not! Perhaps it does some sort of sanity check during tests? Seems weird though...